tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/RuntimeBrokerTests.cs

```diff
@@ -223,6 +243,47 @@ await AssertException.TaskThrowsAsync<MsalUiRequiredException>(
                 .ConfigureAwait(false);
         }
 
+        [IgnoreOnOneBranch]
+        [TestMethod]
+        public async Task WamWithSSHCertificateAuthenticationSchemeAsync()
```

Comment:
>WamWithSSHCertificateAuthenticationSchemeAsync [](http://example.com/codeflow?start=26&length=46)

Thank you for adding this test coverage. Does these tests run in CI ?

Reply:
Yes